### DISCUSSION TITLE
Does anyone know what does this actually install?

### DISCUSSION THREAD

**Discussion by GitExploraa** [YerongAI/Office-Tool#656] [Q&A] [answered]
What does the Bing entry actually install? i tried it but nothing seemed to change

![Screenshot 2023-04-06 235218](https://user-images.githubusercontent.com/44654056/230672974-6b4a43c7-999a-41b0-a466-0bff5e4efdb5.png)

**Comment by YerongAI** [comment, marked as answer]
See https://help.coolhub.top/deploy/basic-settings.html#applications

**Reply by GitExploraa** [reply]
Thank you!
